include/LLGL/Throw.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Throw.h
+ *
+ * Copyright (c) 2015 Lukas Hermanns. All rights reserved.
+ * Licensed under the terms of the BSD 3-Clause license (see LICENSE.txt).
+ */
+
+#ifndef LLGL_THROW_H
+#define LLGL_THROW_H
+
+#if __EXCEPTIONS || __cpp_exceptions == 199711
+    #define LLGL_COMPILER_EXCEPTIONS_ENABLED 1
+#endif
+
+#if (LLGL_DEBUG || LLGL_ENABLE_EXCEPTIONS) && LLGL_COMPILER_EXCEPTIONS_ENABLED
```

Comment:
I still don't think `LLGL_DEBUG` makes sense here. You might want a debug build but turn off exceptions entirely.